packages/headless/src/features/result-preview/result-preview-insight-analytics-actions.ts

```diff
@@ -23,7 +23,7 @@ export const logDocumentQuickview = (result: Result): ClickAction => {
     analyticsPayloadBuilder: (state): InsightPanel.ItemAction => {
       const information = partialDocumentInformation(result, state);
       return {
-        action: 'preview',
+        action: 'preview' as unknown as InsightPanel.ItemAction['action'],
```

Comment:
Why is this needed ?

Reply:
doesn't build without, lemme try it again